src/errors/BadRequestsError.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+// eslint-disable-next-line import/prefer-default-export
+export class BadRequestError extends Error {
```

Comment:
I don't think this error is needed. See my other comment https://github.com/awslabs/fhir-works-on-aws-routing/pull/8#discussion_r485111330

Reply:
Good idea